pkg/services/ngalert/api/test-data/silencing.http

```diff
@@ -0,0 +1,187 @@
+    },
+    "template": []
+  }
+}
+
+### Now let's verify it's there.
+GET http://admin:admin@localhost:3000/alertmanager/grafana/config/api/v1/alerts
+
+###  Let's create some rules for evaluation. If you need to update, include the "uid" as part of the "grafana_alert" key.
+POST http://admin:admin@localhost:3000/ruler/grafana/api/v1/rules/demo
+content-type: application/json
+
+{
+  "name": "demo-alerts-for-demo",
+  "interval": "10s",
+  "rules": [
+    {
+      "grafana_alert": {
+        "title": "DemoAlert",
+        "condition": "condition",
+        "data": [
+          {
+            "refId": "A",
+            "queryType": "",
+            "relativeTimeRange": {
+              "from": 300,
+              "to": 0
+            },
+            "model": {
+              "datasource": "gdev-prometheus",
+              "datasourceUid": "<CHANGE-ME-TO-BE-THE-UID>",
+              "expr": "sum by (handler, job, instance) (http_requests_total)",
+              "hide": false,
+              "interval": "",
+              "intervalMs": 1000,
+              "legendFormat": "",
+              "maxDataPoints": 100,
+              "refId": "A"
+            }
+          },
+          {
+            "refId": "reduced",
+            "queryType": "",
+            "relativeTimeRange": {
+              "from": 18000,
+              "to": 10800
+            },
+            "model": {
+              "datasource": "__expr__",
+              "datasourceUid": "-100",
+              "expression": "A",
+              "hide": false,
+              "intervalMs": 1000,
+              "maxDataPoints": 100,
+              "reducer": "mean",
+              "refId": "reduced",
+              "type": "reduce"
+            }
+          },
+          {
+            "refId": "condition",
+            "queryType": "",
+            "relativeTimeRange": {
+              "from": 18000,
+              "to": 10800
+            },
+            "model": {
+              "datasource": "__expr__",
+              "datasourceUid": "-100",
+              "expression": "$reduced < 10",
+              "hide": false,
+              "intervalMs": 1000,
+              "maxDataPoints": 100,
+              "refId": "condition",
+              "type": "math"
+            }
+          }
+            ]
+      }
+    }
+  ]
+}
+
+### Then, let's make sure our rules are there.
+GET http://admin:admin@localhost:3000/ruler/grafana/api/v1/rules
+
+### While that runs, let's play around with silences.
+### Create a Silence
+POST http://admin:admin@localhost:3000/alertmanager/grafana/api/v2/silences
+content-type: application/json
+
+{
+  "comment": "alerting squad +1",
+  "createdBy": "josh",
+  "startsAt": "2021-03-31T13:17:04.419Z",
+  "endsAt": "2023-03-31T14:17:04.419Z",
+  "matchers": [
+    {
+      "isRegex": true,
+      "name": "instance",
+      "value": "prometheus"
+    }
+  ]
+}
+### Show it to me
+GET http://admin:admin@localhost:3000/alertmanager/grafana/api/v2/silence/<CHANGE-ME>
```

Comment:
```suggestion
GET http://admin:admin@localhost:3000/api/alertmanager/grafana/api/v2/silence/<CHANGE-ME>
```

You can also [save](https://github.com/grafana/grafana/blob/7032ecce2a50e49d28cf00e8960eb0bcfb5403c2/pkg/services/ngalert/api/test-data/am-grafana-recipient.http#L92) the `id` returned from the previous POST request in a variable and [pass](https://github.com/grafana/grafana/blob/7032ecce2a50e49d28cf00e8960eb0bcfb5403c2/pkg/services/ngalert/api/test-data/am-grafana-recipient.http#L96) this one instead of `<CHANGE-ME>`